kotsadm/pkg/airgap/airgap.go

```diff
@@ -256,7 +256,7 @@ func CreateAppFromAirgap(pendingApp *PendingApp, airgapBundle multipart.File, re
 		return errors.Wrap(err, "failed to update registry")
 	}
 
-	query = `update app set install_state = 'installed', is_airgap=true where id = $1`
```

Comment:
does anything replace this install_state=installed?

Reply:
yep, in here: https://github.com/replicatedhq/kots/blob/eec7cd5aeee617dd72901cb870eed51043f090b8/kotsadm/pkg/airgap/airgap.go#L85